recursion/compiler/src/gnark/lib/main.go

```diff
@@ -16,11 +16,11 @@ func (circuit *Circuit) Define(api frontend.API) error {
 	
 	// Variables.
 	var felt2 *babybear.Variable
-	var felt0 *babybear.Variable
```

Comment:
intended change?

Reply:
Oh this file changes every time we run tests. Maybe we will gitignore it in the future